src/hypothesis/internal/compat.py

```diff
@@ -392,7 +392,7 @@ def __add__(self, value):
         return compatbytes(bytearray.__add__(self, value))
 
     def __radd__(self, value):
-        return compatbytes(bytearray.__radd__(self, value))
+        return compatbytes(bytearray.__add__(value, self))
```

Comment:
compat.py is excluded from the coverage checks because there are large swathes of it that only get invoked on one version of Python or another and we only run coverage on Python 3.

We never spotted the non-existence of bytearray.__radd__ because this method was never invoked. We only ever add hbytes to hbytes (and don't expose hbytes to end users, thank the gods and/or past-DRMacIver), so this always goes through \_\_add\_\_ and \_\_radd\_\_ is never invoked. I only found this because I accidentally did str + hbytes, which goes via hbytes.\_\_radd\_\_ and triggers this problem.

Basically this method can only get invoked when we do something wrong.